### ISSUE TITLE
Should you be able to focus on a button that is disabled?

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#317]
In https://github.com/phetsims/reactants-products-and-leftovers/issues/56 I realized that when a button is disabled via the `enabledProperty`, it is still focusable with a11y. @jessegreenberg how is the best way to tackle this. I don't think we want this functionality.

**Comment by jessegreenberg** [comment]
Good question @zepumph, I am not sure.  I have read that it is important for a11y to keep disabled buttons focusable so that AT users can still find them and so the flow remains the same. AT read these buttons as "Disabled". (https://ux.stackexchange.com/questions/103239/should-disabled-elements-be-focusable-for-accessibility-purposes)

But it looks like Chrome removes disabled buttons by default! (https://jsfiddle.net/pLuk8th0/)

@terracoda do you have any thoughts about this?

**Comment by jessegreenberg** [comment]
W3C recommendation is pretty vague: https://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled.

> Disabled elements might not receive focus from the tab order.

**Comment by jessegreenberg** [comment]
Now I am leaning toward removing these from focus order, from [MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Attribute/disabled):

> If the element is disabled, it does not respond to user actions, it cannot be focused, and the command event will not fire.

But it would still be good to hear from @terracoda.

**Comment by terracoda** [comment]
@zepumph, I will look into this, but my intuition is that a control that is disabled and visible should be findable and communicated as disabled to AT users. Need to check about the actual findability level (remains in Tab order, or remains findable with cursor keys).

**Comment by jessegreenberg** [comment]
@terracoda in a discussion today with @emily-phet and @zepumph we decided that it was best to keep disabled buttons in the navigation order like your original intuition. @zepumph brought up that that for groups of UI components that can be enabled/disabled it is much more convenient to remember how many buttons are in the order and leverage that to navigate the page quickly. In that sense it is *less* convenient for the user to remove them. Since the W3C recommendation lets us chose, we are going to proceed with this. Closing issue, but let us know if you have concerns.

**Comment by terracoda** [comment]
@jessegreenberg, sounds good. Sorry I didn't get back to you on this issue.
